shell/components/CopyToClipboardText.vue

```diff
@@ -39,17 +49,38 @@ export default {
 
 <template>
   <a
+    v-tooltip="toolTip ? {content: 'Copy value'} : {content: null}"
     class="copy-to-clipboard-text"
     :class="{ 'copied': copied, 'plain': plain}"
     href="#"
     @click="clicked"
   >
-    {{ text }} <i
-      class="icon"
-      :class="{ 'icon-copy': !copied, 'icon-checkmark': copied}"
-    />
+    <span v-if="showLabel">{{ text }}</span>
+    <v-popover
+      popover-class="clipboard-text-copied"
+      placement="top"
+      :trigger="toolTip ? 'click' : ''"
+      :open="copied && toolTip"
+    >
+      <i
+        class="icon"
+        :class="{ 'icon-copy': !copied, 'icon-checkmark': copied}"
+      />
+      <template slot="popover">
+        <span>Value copied!</span>
+      </template>
+    </v-popover>
   </a>
 </template>
+<style lang="scss">
```

Comment:
We should not have customization of this vendor in a specific component. As far as I can see, we have a [global styling for this](https://github.com/rancher/dashboard/blob/b092f25bfdae65586948e024a06bfbf5e0502b00/shell/assets/styles/global/_tooltip.scss). Could not find the library or component import either, so I could not give further informations.

Reply:
As agreed, global use cases are overlapping due different scopes and usages.

To be tackled with https://github.com/rancher/dashboard/issues/9850